integrations/dns_data.go

```diff
@@ -0,0 +1,162 @@
+package integrations
+
+import (
+	"errors"
+	cf "github.com/cloudflare/cloudflare-go"
+	"time"
+)
+
+type DNSResource struct{
+	Zone string
+	Name string
+	cfRecordId string
+	cfZoneId string
+	cfProxied bool
+	cfModifiedOn time.Time
+}
+
+func (f DNSResource)FQDN()string{
```

Comment:
Codacy found an issue: [exported method DNSResource.FQDN should have comment or be unexported](https://app.codacy.com/gh/go-go-dojo/friendly-potato/pullRequest?prid=6165294)